notebooks/MAST/downloading_transiting_planet_data/downloading_exoplanet_data.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<p>As a reminder the <a href="https://sciencebetter.slack.com/archives/C03FDR44L07/p1660233388139449" target="_blank">new template </a>includes a Workflow section. Basically a Table of Contents that will match the header levels of the sections that follow.</p><p> </p><p>Also I'd recommend, based on the template, to reword each section header below with more descriptive headers. I'll try to make recommendations as I go. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>